src/ateles.erl

```diff
@@ -123,14 +125,35 @@ validate_doc_update(DDoc, EditDoc, DiskDoc, UserCtx, SecObj) ->
     JsonEditDoc = couch_doc:to_json_obj(EditDoc, [revs]),
     JsonDDoc = couch_doc:to_json_obj(DDoc, []),
     JsonDiskDoc = couch_query_servers:json_doc(DiskDoc),
-    ddoc_prompt(JsonDDoc, <<"validate_doc_update">>, [JsonEditDoc, JsonDiskDoc, UserCtx, SecObj]).
+    ddoc_prompt(JsonDDoc, <<"validate">>, [<<"validate_doc_update">>], [JsonEditDoc, JsonDiskDoc, UserCtx, SecObj]).
 
-ddoc_prompt({Props} = DDoc, FunName, Args) ->
+filter_view(DDoc, VName, Docs) ->
+    JsonDocs = [couch_query_servers:json_doc(Doc) || Doc <- Docs],
+    JsonDDoc = couch_doc:to_json_obj(DDoc, []),
```

Comment:
`couch_js` based version of the feature uses `Options` to  pass `revs_limit` [here](https://github.com/apache/couchdb/blob/main/src/couch_js/src/couch_js_query_servers.erl#L443:L444). Is it no longer  needed?

Reply:
I couldn't figure out why it would be needed. But thinking now it could be to limit the size of the document?

Reply:
Here is  where  it  was  first   introduced https://github.com/apache/couchdb/commit/23e5f92a15cdc868622481d2ff63ca3770110fae

Reply:
I've added in the `Options` params.